eve/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+version: 0.2
+
+branches:
+  default:
+    stage: pre-merge
+
+stages:
+  pre-merge:
+    worker: &master-worker
+      type: docker
+      path: eve/workers/master
+      volumes:
+        - '/home/eve/workspace'
+    steps:
+      - Git:
+          name: fetch source
+          repourl: '%(prop:git_reference)s'
+          shallow: 'True'
+          retryFetch: 'True'
+          haltOnFailure: 'True'
+      - ShellCommand:
+          name: install dependencies
+          command: npm install
+      - ShellCommand:
+          name: run lint yml
+          command: npm run --silent lint_yml
+      - ShellCommand:
+          name: run lint
+          command: npm run --silent lint -- --max-warnings 0
+      - ShellCommand:
+          name: run lint_md
+          command: npm run --silent lint_md
+      - ShellCommand:
+          name: set host testing.local
+          command: sudo bash -c 'echo "127.0.0.1 testing.local" >> /etc/hosts'
+      - ShellCommand:
+          name: set host vaultclient.testing.local
+          command: sudo bash -c \
```

Comment:
not asking to change anything but in case you didn't knew.
you can use the yaml syntax of multiline in ShellCommand.
Example:
```yaml
       - ShellCommand:
           name: set host vaultclient.testing.local
           command: |
             sudo bash -c 'echo "127.0.0.1 vault.testing.local" >> /etc/hosts'
```
I don't know if this pass the yaml lint. Try it if you want to.

Reply:
Tried on it's working . Thanks :)